Dockerfile

```diff
@@ -11,6 +11,10 @@ ENV LANG C.UTF-8
 
 RUN apt-get update && apt-get install -y gpg curl wget unzip xz-utils git openssh-client && apt-get clean -y
 
+## Gradle
+
+RUN apt-get update && apt-get install -y --no-install-recommends openjdk-8-jdk gradle && apt-get clean -y
```

Comment:
Should we model this after https://github.com/renovatebot/docker-gradle ?

Reply:
I added this so that you use renovate via docker (`docker run renovate/renovate`), gradle works. Otherwise you have to use `docker run renovate/renovate --binary-source=docker` to make it works. 

I believe that it should work out of the box, thus the change.

Reply:
I think it would be ok having the jdk installed, so it can be used with gradlew, but if gradle has its own dockerfile, it should not be installed here as well.

Reply:
but this dockerfile is the one used to build `renovate/renovate` image right? 

Maybe I am wrong here but you need to have gradle installed within it so that gradle also works directly when using renovate docker image? The gradle dockerfile is only used when you use `--binary-source=docker`, isn't it?